tests/ui/feature-gates/feature-gate-adt_const_params.stderr

```diff
@@ -5,14 +5,6 @@ LL | struct Foo<const NAME: &'static str>;
    |                        ^^^^^^^^^^^^
    |
    = note: the only supported types are integers, `bool`, and `char`
-help: add `#![feature(adt_const_params)]` to the crate attributes to enable more complex and user defined types
```

Comment:
driveby: can you change this test to use a const generic parameter that would actually be supported under `adt_const_params. e.g. `struct Foo<const N: Bar>; struct Bar(u8);`